### ISSUE TITLE
Manual captcha stats on bookmarklet all 0 after solving

### ISSUE THREAD

**Issue by jtp10181** [RocketMap/RocketMap#1804]
## Expected Behavior

Expect after solving a captcha the page would refresh and display current stats

## Current Behavior

Page refreshes and displays all 0 for the stats after solving.
If I click the bookmark again it loads correctly.

I think if I solve without the stats there, it doesn't seem to actually work (cannot 100% confirm)

## Steps to Reproduce (for bugs)

1. Run server with manual captcha
2. Solve a captcha
3. Observe stats at bottom

## Context

Annoying to have to keep clicking the bookmark after every solve.

## Your Environment

* Version used: Current Pull https://github.com/RocketMap/RocketMap/commit/4471f748b83f234d465815f58439956f8e5dedaf
* Environment name and version (e.g. Python 2.7): Python 2.7
* Operating System and version (desktop or mobile): WIndows 7 x64

**Comment by neskk** [comment]
Try to debug with some logs perhaps? 
This problem happens when you don't have `-sn`/`--status-name` defined and so the statistics won't be uploaded to database.

**Comment by walaoaaa1234** [comment]
i added a line after
```
setTimeout(initCaptchaPage, 1500);
setTimeout(refreshStats, 1500);
```

**Comment by myntath** [comment]
Can confirm I have the same issue and walaoaaa1234's fix worked for me.

Can confirm I do have a status-name set. Statistics work on first load and but revert to all 0's after solving every captcha. the 'get_stats' page is working fine so definitely an inject.js issue.

**Comment by jtp10181** [comment]
Yes I have the status name set, always do. What file to patch for the fix above? I can try it out also.

**Comment by neskk** [comment]
@walaoaaa1234 and @jtp10181 I going to get this fixed soon and it will also include a fix to prevent duplicate setInterval instances. (timer going too fast)
I understood that no statistics were ever presented, if this were the case, enabling `-sn` would have been the solution. It's a very quick fix in "templates/inject.js".

**Comment by neskk** [comment]
https://github.com/RocketMap/RocketMap/pull/1796

Bugs fixed.

**Comment by jtp10181** [comment]
I tested the first fix above and its working. Will wait for the PR to get merged for the full set of fixes.
@neskk nice work on this whole captcha setup.

**Comment by sebastienvercammen** [comment]
Fixed in #1796.
